src/core/models/post.php

```diff
@@ -10,7 +10,7 @@ static function getById($id)
 
   static function getByIdSlug($id)
   {
-    $db = \gila::slaveDB();
+    $db = \Gila::slaveDB();
```

Comment:
You could do `use Gila` and remove the namespace definition.

Reply:
That's cool